crates/libtiny_tui/src/msg_area/mod.rs

```diff
@@ -67,9 +67,14 @@ impl MsgArea {
     }
 
     pub(crate) fn resize(&mut self, width: i32, height: i32) {
-        self.width = width;
+        self.width = if self.scroll != 0 { width - 1 } else { width };
```

Comment:
How does this work? Why do you make the area smaller in width if not scrolled to the bottom?

Reply:
This was a merge error. I have a lot of crazy branches going and trying to sync them! I will remove this.

FYI it was for making room for a scrollbar